qt/widgets/common/src/MWRunFiles.cpp

```diff
@@ -524,15 +524,18 @@ void MWRunFiles::findFiles(bool isModified) {
   const auto directory = findFilesDirectory(searchText);
 
   // Set the values for the thread, and start it running.
-  if (directory)
-    m_foundFiles.append(directory.get());
-  else {
+  if (directory) {
```

Comment:
This feels a little ugly to be honest. Are we gaining much in clarity by refactoring that small bit of directory handling code?

I absolutely agree that the part handling the instrument name below needed refactoring and now that has been done maybe it's not too bad leaving the ~10 lines from `findFilesDirectory` in here directly?